scripts/sync-labels.sh

```
@@ -0,0 +1,80 @@
+
+echo
+echo "reconciled $declared label(s) from $SOURCE"
+
+# What the repository has and the file does not name. Reported, never removed.
+extra="$(comm -13 \
+  <(awk '/^- name: "/ { n = $0; sub(/^- name: "/, "", n); sub(/"$/, "", n); print n }' "$SOURCE" | sort) \
+  <(gh label list --limit 200 --json name --jq '.[].name' | sort))"
+if [[ -n "$extra" ]]; then
+  echo
+  echo "NOT IN $SOURCE, and left alone:"
+  printf '  %s\n' $extra
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)